libraries/Wire/Wire.h

```diff
@@ -28,9 +28,7 @@
 
 #include "RingBuffer.h"
 
-#define BUFFER_LENGTH 32
```

Comment:
it's not work yet
I2Cdev come from https://github.com/jrowberg/i2cdevlib
it's provided nrf51 version but it's not for this i think.
i change this lib used nrf51 version
when i compile code it's error: nrf_drv_twi.h: No such file or directory
get this nrf_drv _twi.h will fix this error?
i am beginer thanks